scyjava/convert/_convert.py

```diff
@@ -326,6 +334,9 @@ def to_python(data):
     if StringClass.isInstance(data):
         return data.toString()
 
+    if jclass('org.scijava.table.Table').isInstance(data):
```

Comment:
Here I call `jclass('org.scijava.table.Table')` inside the function to avoid having an error when the `scyjava` package is imported without having `org.scijava.table.Table` in the classpath.